lib/ae_mdw/db/format.ex

```diff
@@ -117,7 +117,7 @@ defmodule AeMdw.Db.Format do
   end
 
   defp custom_raw_data(state, :contract_call_tx, tx, tx_rec, signed_tx, block_hash) do
-    contract_pk = :aect_call_tx.contract_pubkey(tx_rec)
+    {_id_tag, contract_pk} = tx_rec |> :aect_call_tx.contract_id() |> :aeser_id.specialize()
```

Comment:
Maybe we could have a helper function to get only the desired data since the `id_tag` is ignored in all places.

Reply:
Good call, added `Db.id_pubkey/1` to access the pubkey from the aeser id